src/usr/share/gpgmailer/gpgmailmessage.py

```diff
@@ -67,38 +62,17 @@ class GpgMailMessage:
     Note: Each method should check if this object has already been saved and
       throw an exception if it has.
     """
-    _outbox_dir = None
-    _draft_dir = None
-
-    # TODO: Eventually make this method so it can be called twice.
-    @classmethod
-    def configure(cls):
-        """Reads the gpgmailer config file to obtain the watch directory's path name.
-        This method must be called before any instances are created.
-        """
-        logger = logging.getLogger('GpgMailMessage')
-
-        config_file = ConfigParser.SafeConfigParser()
-        config_file.read('/etc/gpgmailer/gpgmailer.conf')
-
-        config_helper = confighelper.ConfigHelper()
-
-        mail_dir = config_helper.verify_string_exists(config_file, 'watch_dir')
-        cls._outbox_dir = os.path.join(mail_dir, 'outbox')
-        cls._draft_dir = os.path.join(mail_dir, 'draft')
-
-        if not(os.path.isdir(cls._outbox_dir)) or not(os.path.isdir(cls._draft_dir)):
-            logger.critical('A watch subdirectory does not exist. Quitting.')
-            raise WatchDirectoryMissingException('A watch subdirectory does not exist.')
 
     def __init__(self):
         """Initializes the class."""
 
-        # Verify the 'configure' class method was called.
-        if (self._outbox_dir is None) or (self._draft_dir is None):
-            # TODO: Consider just calling configure here instead of raising an exception.
-            raise GpgMailMessageNotConfiguredException(
-                'GpgMailMessage.configure() must be called an instance can be created.')
+        logger = logging.getLogger('GpgMailMessage')
```

Comment:
The logger naming convention is `getLogger(__name__)` instead of a string literal. That way if we change the class name, we won't have to worry about changing this literal, too.

Reply:
Fine!!!